devicetypes/vision-raytseng/vision-4-in-1-motion-sensor.src/vision-4-in-1-motion-sensor.groovy

```diff
@@ -0,0 +1,552 @@
+/**
+ *  Vision 4-in-1 Motion Sensor
+ *
+ *  Author: Ray Tseng
+ */
+metadata {
+  definition (name: "Vision 4-in-1 Motion Sensor", namespace: "vision-raytseng", author: "Ray Tseng", vid: "generic-motion-8", ocfDeviceType: "x.com.st.d.sensor.motion") {
+    capability "Battery"
+    capability "Motion Sensor"
+    capability "Relative Humidity Measurement"
+    capability "Temperature Measurement"
+    capability "Illuminance Measurement"
+    capability "Tamper Alert"
+    capability "Health Check"
+    
+    fingerprint mfr:"0109", prod:"2021", model:"2112", deviceJoinName: "Vision ZP3113-7 4-in-1 Motion"
```

Comment:
Please change deviceJoinName to "Vision Multipurpose Sensor" in this fingerprint and add Raw Description in a comment after the fingerprint so it would look like:
fingerprint mfr:"0109", prod:"2021", model:"2112", deviceJoinName: "Vision Multipurpose Sensor" // Raw Description // Raw Description >>paste the raw description here<<

(Raw description can be found at the Smartthings graphapi in device details)

Reply:
What about the Raw Description in a comment after the fingerprint ?